net/usrsock/usrsock_sendmsg.c

```diff
@@ -71,7 +71,8 @@ static uint16_t sendto_event(FAR struct net_driver_s *dev,
 
       pstate->result = conn->resp.result;
 
-      if (pstate->result >= 0 || pstate->result == -EAGAIN)
+      if (!(flags & USRSOCK_EVENT_SENDTO_READY) &&
```

Comment:
Optional
```suggestion
      if ((flags & USRSOCK_EVENT_SENDTO_READY) == 0 &&
```

Reply:
@pkarashchenko there are many places in both files use `!(flags & xxx)` in if statement, so I prefer to keep the change as it.

Reply:
NP. It is optional.